package.json

```
@@ -25,7 +25,9 @@
     "integration:single": "testcafe chrome -L",
     "integration:headless:prod": "testcafe --config-file \".testcaferc-prod.json\"",
     "integration:headless:dev": "testcafe --config-file \".testcaferc-dev.json\"",
-    "integration:metamask": "run-s wait blockchain:contracts integration:synpress",
+    "integration:testcafe": "run-s blockchain:contracts:testcafe integration:headless:prod",
```

```suggestion
    "integration:testcafe": "run-s blockchain:contracts:testcafe integration:testcafe:prod",
```
to align with the renames?

---

ok sure